### PR TITLE
bump version, fix import test

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
@pmlandwehr is this good to merge?

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
this doesn't build in windows py35 or greater because of encoding issues in the source setup.py file.

I proposed a change to the source repo, which fixes it. We'll see what they do about it.

**Comment by agcopenhaver** [comment]
@conda-forge/core I don't know if it's too soon to ping you on this, but it doesn't look like this repo gets a ton of attention

**Comment by mariusvniekerk** [comment]
@agcopenhaver feel free to add yourself as a maintainer.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,40 +1,46 @@
...
 
 about:
-  home: https://github.com/mattgwwalker/msg-extractor
-  license: GPL 3.0
-  license_file: LICENSE.txt
-  license_family: GPL3
+  home: "https://github.com/mattgwwalker/msg-extractor"
+  license: "GPL"
+  license_family: "GPL"
+  license_file: ""
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: This has to be filled.   Not distributing the license for GPL packages is a problem.

Comment (agcopenhaver) [reply recipe/meta.yaml]: Wow - that was dumb of me. Sorry for the sloppiness.

**Comment by scopatz** [comment]
@conda-forge-admin please rerender

**Pull request merged**